temp_guide_to_setup_a_node_validator.md

```diff
@@ -0,0 +1,50 @@
+# Setting Up a Node/Validator for Paseo Testnet
+
+Welcome to the Paseo testnet! This guide will walk you through the process of setting up a node and becoming a validator on the Paseo testnet network using the Polkadot binary, which is also used for Polkadot and Kusama validators.
+
+## Prerequisites
+
+Before you begin, make sure you have the following:
+
+- A server with a recommended minimum of 8GB RAM, a quad-core CPU, and at least 200GB of free disk space.
```

Comment:
I do not remember which were the specs, if they are this perfect 👍🏻

Reply:
Actually no, these specs are not right. We have a PAS that i think we could refer here:

https://github.com/paseo-network/paseo-action-submission/blob/main/pas/Hardware_specs.md

Reply:
Let's change them 👍🏻

Reply:
already updated @hbulgarini @carlosala please take a look 🙏